src/controller.h

```diff
@@ -57,6 +57,7 @@ class Controller : public QObject, public LogSerializer {
   void updateRequired();
   void deleteOSTunnelConfig();
   void startHandshakeTimer();
+  bool deviceNetworkConnectivity() const { return m_deviceNetworkConnectivity; }
```

Comment:
What is the intended use of this method? (Also, consider renaming it to something like isDeviceConnected, isNetworkAvailable etc.)